services/monitors/monitors/parsers.go

```diff
@@ -91,27 +69,10 @@ func (monitor *Monitor) FromMap(monitorMap map[string]any) error {
 	return nil
 }
 
-func (statistics *Statistics) FromMap(statisticsMap map[string]any) error {
-	if len(statisticsMap["last_evaluation"].([]any)) > 0 {
-		if err := statistics.LastEvaluation.FromMap(statisticsMap["last_evaluation"].([]any)[0].(map[string]any)); err != nil {
-			return err
-		}
-	}
-	return nil
-}
-
-func (evaluation *Evaluation) FromMap(evaluationMap map[string]any) error {
-	evaluation.Timestamp = evaluationMap["timestamp"].(string)
-	evaluation.Value = evaluationMap["value"].(float64)
-	evaluation.Status = evaluationMap["status"].(string)
-	return nil
-}
-
-func (expression *Expression) FromMap(expressionMap map[string]any) error {
-	expression.ComparisonOperator = expressionMap["comparison_operator"].(string)
-	expression.ComparisonValue = expressionMap["comparison_value"].(float64)
-	expression.AggregationFunction = expressionMap["aggregation_function"].(string)
-	expression.Tolerance = expressionMap["tolerance"].(float64)
+func (rule *Rule) FromMap(expressionMap map[string]any) error {
```

Comment:
Here too, can you change like that?
```suggestion
func (rule *Rule) FromMap(ruleMap map[string]any) error {
```